### ISSUE TITLE
Flutter Application freezes when Maximized or Restored from Minimized window mode

### ISSUE THREAD

**Issue by AldrinMathew** [flutter/flutter#74797]
## Steps to Reproduce

1. Run **flutterm create bug** or _**flutter create bug**_ after running _flutter channel master_ switching to the master channel.
2. Run **flutterm run -d windows**
3. Alternatively, in VS Code, select the _Windows_ device option. Press F5 or press the Play Icon that runs the app without debugging.
4. The default startup app of flutter will show up after a few seconds. Minimize the App and then Maximize/Restore it by clicking on the taskbar icon. Alternatively, press _Alt + Tab_ to switch to the App window after minimizing. The Flutter App window will freeze and if you wait for 5 seconds, will switch to the _App not Responding_ style window of Windows OS.

## Note
1. This issue started almost a week ago. That might help in determining which version of the master channel is responsible for this issue. After a lot of rebuilds leading to the problem, I upgraded the master channel on 26/01/2021 from version **1.26.0-2.0.pre.120** to **1.26.0-13.0.pre.179**. The problem persisted after upgrading.
2. You will see the commands starting with _**flutterm**_. That is just an Alias I set up for the master channel of flutter. Aliases are used to manage different versions of flutter in the same machine. I also have the Stable (flutter) and Dev (flutterd) channels of flutter setup in my machine. **The Flutter Master (flutterm) channel is used for other apps in the same machine. This problem DOES OCCUR in those applications**. I restarted my machine, run the app and it happened again. I removed the _Debug_ and _Release_ folders in the following folders and built the app again to see if the problem persists. It does.
   - build\windows\flutter
   - build\windows\flutter/x64
   - build\windows\runner
   - build\windows\runner\helloproject.dir
   - build\windows\flutter_wrapper_app.dir
   - build\windows\flutter_wrapper_plugin.dir
   - build\windows\x64

**Expected results:** 

The Flutter window is supposed to behave like any normal window on the installed system. The minimize, maximize, restore, close functionality should work like in any other window.

**Actual results:** 

The Flutter channel is _master_. The platform is Windows 10. Whenever the app starts, it works fine. The functionality of the app that I am developing works just like I intended it to be. But then whenever I minimize the Flutter App's window and maximizes or restores again, **the App freezes**. No new frames are rendered and the Flutter window becomes unresponsive. However, this problem does not occur when I switch between maximize and restore window modes. **Only happens when I go from Minimized to Maximize or Restore.**

**Also in Release Build:**
This problem is seen when I execute the command _**flutterm build windows**_. I opened the .exe file generated, but the same problem occurs. The App freezes after I minimize the Window and restores, or maximizes it after that.

## Logs
<details>
  <summary>Flutter Run in Verbose</summary>

```
flutterm run --verbose
[  +86 ms] executing: [C:\src\flutter_master/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[  +73 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +5 ms] fc9addb88b5262ce98e8f39b0eefa6fa9be2ca6a
[   +1 ms] executing: [C:\src\flutter_master/] git tag --points-at fc9addb88b5262ce98e8f39b0eefa6fa9be2ca6a
[  +50 ms] Exit code 0 from: git tag --points-at fc9addb88b5262ce98e8f39b0eefa6fa9be2ca6a
[   +2 ms] executing: [C:\src\flutter_master/] git describe --match *.*.* --long --tags fc9addb88b5262ce98e8f39b0eefa6fa9be2ca6a
[  +66 ms] Exit code 0 from: git describe --match *.*.* --long --tags fc9addb88b5262ce98e8f39b0eefa6fa9be2ca6a
[        ] 1.26.0-12.0.pre-179-gfc9addb88b
[  +58 ms] executing: [C:\src\flutter_master/] git rev-parse --abbrev-ref --symbolic @{u}
[  +38 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/master
[        ] executing: [C:\src\flutter_master/] git ls-remote --get-url origin
[  +34 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[  +81 ms] executing: [C:\src\flutter_master/] git rev-parse --abbrev-ref HEAD
[  +37 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] master
[ +105 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[  +14 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +71 ms] executing: C:\Users\aldri\AppData\Local\Android\sdk\platform-tools\adb.exe devices -l
[  +49 ms] List of devices attached
[   +6 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +5 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +6 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[  +10 ms] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +94 ms] Multiple devices found:
[  +26 ms] Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19042.746]
[   +1 ms] Chrome (web)      • chrome  • web-javascript • Google Chrome 87.0.4280.141
[        ] Edge (web)        • edge    • web-javascript • Microsoft Edge 88.0.705.50
[   +2 ms] [1]: Windows (windows)
[        ] [2]: Chrome (chrome)
[        ] [3]: Edge (edge)
[   +2 ms] Please choose one (To quit, press "q/Q")
[        ] : 
[+1102 ms] 1
[  +17 ms] Skipping pub get: version match.
[ +175 ms] Generating D:\projects\helloproject\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java
[ +135 ms] Initializing file store
[  +13 ms] Skipping target: gen_localizations
[   +9 ms] complete
[   +5 ms] Launching lib\main.dart on Windows in debug mode...
[  +15 ms] C:\src\flutter_master\bin\cache\dart-sdk\bin\dart.exe --disable-dart-dev
C:\src\flutter_master\bin\cache\artifacts\engine\windows-x64\frontend_server.dart.snapshot --sdk-root
C:\src\flutter_master\bin\cache\artifacts\engine\common\flutter_patched_sdk/ --incremental --target=flutter --debugger-module-names
--experimental-emit-debug-metadata -DFLUTTER_WEB_AUTO_DETECT=true --output-dill
C:\Users\aldri\AppData\Local\Temp\flutter_tools.1d88acec\flutter_tool.dc6d9895\app.dill --packages
D:\projects\helloproject\.dart_tool\package_config.json -Ddart.vm.profile=false -Ddart.vm.product=false --enable-asserts --track-widget-creation 
--filesystem-scheme org-dartlang-root --initialize-from-dill build\f245b0b7cc30f460f818fcec6bcf01fb.cache.dill.track.dill --flutter-widget-cache 
--enable-experiment=alternative-invalidation-strategy
[  +41 ms] executing: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version 16
-requires Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64
Microsoft.VisualStudio.Component.VC.CMake.Project
[  +38 ms] Exit code 0 from: C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe -format json -products * -utf8 -latest -version16 -requires Microsoft.VisualStudio.Workload.NativeDesktop Microsoft.VisualStudio.Component.VC.Tools.x86.x64
Microsoft.VisualStudio.Component.VC.CMake.Project
[   +1 ms] [
             {
               "instanceId": "3d840ec5",
               "installDate": "2020-12-04T12:55:34Z",
               "installationName": "VisualStudio/16.8.4+30907.101",
               "installationPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community",
               "installationVersion": "16.8.30907.101",
               "productId": "Microsoft.VisualStudio.Product.Community",
               "productPath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\Common7\\IDE\\devenv.exe",
               "state": 4294967295,
               "isComplete": true,
               "isLaunchable": true,
               "isPrerelease": false,
               "isRebootRequired": false,
               "displayName": "Visual Studio Community 2019",
               "description": "Powerful IDE, free for students, open-source contributors, and individuals",
               "channelId": "VisualStudio.16.Release",
               "channelUri": "https://aka.ms/vs/16/release/channel",
               "enginePath": "C:\\Program Files (x86)\\Microsoft Visual
               Studio\\Installer\\resources\\app\\ServiceHub\\Services\\Microsoft.VisualStudio.Setup.Service",
               "releaseNotes": "https://go.microsoft.com/fwlink/?LinkId=660893#16.8.4",
               "thirdPartyNotices": "https://go.microsoft.com/fwlink/?LinkId=660909",
               "updateDate": "2021-01-20T04:11:08.4395Z",
               "catalog": {
                 "buildBranch": "d16.8",
                 "buildVersion": "16.8.30907.101",
                 "id": "VisualStudio/16.8.4+30907.101",
                 "localBuild": "build-lab",
                 "manifestName": "VisualStudio",
                 "manifestType": "installer",
                 "productDisplayVersion": "16.8.4",
                 "productLine": "Dev16",
                 "productLineVersion": "2019",
                 "productMilestone": "RTW",
                 "productMilestoneIsPreRelease": "False",
                 "productName": "Visual Studio",
                 "productPatchVersion": "4",
                 "productPreReleaseMilestoneSuffix": "1.0",
                 "productSemanticVersion": "16.8.4+30907.101",
                 "requiredEngineVersion": "2.8.3267.30329"
               },
               "properties": {
                 "campaignId": "",
                 "channelManifestId": "VisualStudio.16.Release/16.8.4+30907.101",
                 "nickname": "",
                 "setupEngineFilePath": "C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\vs_installershell.exe"
               }
             }
           ]
[  +37 ms] Building Windows application...
[  +25 ms] <- compile package:helloproject/main.dart
[   +3 ms] executing: C:\Program Files (x86)\Microsoft Visual
Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe -S D:\projects\helloproject\windows -B build\windows -G   
Visual Studio 16 2019
[  +60 ms] -- Selecting Windows SDK version 10.0.18362.0 to target Windows 10.0.19042.
[  +42 ms] -- Configuring done
[ +251 ms] -- Generating done
[  +24 ms] -- Build files have been written to: D:/projects/helloproject/build/windows
[   +9 ms] executing: C:\Program Files (x86)\Microsoft Visual
Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe --build build\windows --config Debug --target INSTALL     
--verbose
[ +182 ms] Microsoft (R) Build Engine version 16.8.3+39993bd9d for .NET Framework
[   +1 ms] Copyright (C) Microsoft Corporation. All rights reserved.
[  +73 ms] Build started 26-01-2021 22:48:00.
[ +345 ms] Project "D:\projects\helloproject\build\windows\INSTALL.vcxproj" on node 1 (default targets).
[   +1 ms] Project "D:\projects\helloproject\build\windows\INSTALL.vcxproj" (1) is building
"D:\projects\helloproject\build\windows\ZERO_CHECK.vcxproj" (2) on node 1 (default targets).
[  +17 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +21 ms] CustomBuild:
[   +1 ms]   All outputs are up-to-date.
[  +54 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\ZERO_CHECK\ZERO_CHECK.tlog\ZERO_CHECK.lastbuildstate".
[   +1 ms] Done Building Project "D:\projects\helloproject\build\windows\ZERO_CHECK.vcxproj" (default targets).
[ +130 ms] Project "D:\projects\helloproject\build\windows\INSTALL.vcxproj" (1) is building
"D:\projects\helloproject\build\windows\ALL_BUILD.vcxproj" (3) on node 1 (default targets).
[   +1 ms] Project "D:\projects\helloproject\build\windows\ALL_BUILD.vcxproj" (3) is building
"D:\projects\helloproject\build\windows\flutter\flutter_wrapper_app.vcxproj" (4) on node 1 (default targets).
[   +1 ms] Project "D:\projects\helloproject\build\windows\flutter\flutter_wrapper_app.vcxproj" (4) is building
"D:\projects\helloproject\build\windows\flutter\flutter_assemble.vcxproj" (5) on node 1 (default targets).
[   +1 ms] InitializeBuildStatus:
[        ]   Creating "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +1 ms] CustomBuild:
[        ]   "The build of 'D:\projects\helloproject\build\windows\CMakeFiles\a2c979d5bf1867ebf2bec8313cacd96a\flutter_assemble.rule' depends on 
'D:\PROJECTS\helloproject\WINDOWS\FLUTTER\EPHEMERAL\FLUTTER_WINDOWS.DLL' which is produced by the build of
'D:\projects\helloproject\build\windows\CMakeFiles\48bb0af4f19d34c0935a125b5895e4db\flutter_windows.dll.rule'. The items cannot be built in      
parallel."
[ +103 ms]   Generating D:/projects/helloproject/windows/flutter/ephemeral/flutter_windows.dll,
D:/projects/helloproject/windows/flutter/ephemeral/flutter_export.h, D:/projects/helloproject/windows/flutter/ephemeral/flutter_windows.h,       
D:/projects/helloproject/windows/flutter/ephemeral/flutter_messenger.h,
D:/projects/helloproject/windows/flutter/ephemeral/flutter_plugin_registrar.h,
D:/projects/helloproject/windows/flutter/ephemeral/cpp_client_wrapper/core_implementations.cc,
D:/projects/helloproject/windows/flutter/ephemeral/cpp_client_wrapper/standard_codec.cc,
D:/projects/helloproject/windows/flutter/ephemeral/cpp_client_wrapper/plugin_registrar.cc,
D:/projects/helloproject/windows/flutter/ephemeral/cpp_client_wrapper/flutter_engine.cc,
D:/projects/helloproject/windows/flutter/ephemeral/cpp_client_wrapper/flutter_view_controller.cc, _phony_
[+2023 ms]   [ +123 ms] executing: [C:\src\flutter_master/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[  +83 ms]   [ +126 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +1 ms]   [        ] fc9addb88b5262ce98e8f39b0eefa6fa9be2ca6a
[        ]   [   +1 ms] executing: [C:\src\flutter_master/] git tag --points-at fc9addb88b5262ce98e8f39b0eefa6fa9be2ca6a
[  +90 ms]   [  +90 ms] Exit code 0 from: git tag --points-at fc9addb88b5262ce98e8f39b0eefa6fa9be2ca6a
[   +2 ms]   [   +2 ms] executing: [C:\src\flutter_master/] git describe --match *.*.* --long --tags fc9addb88b5262ce98e8f39b0eefa6fa9be2ca6a
[ +118 ms]   [ +117 ms] Exit code 0 from: git describe --match *.*.* --long --tags fc9addb88b5262ce98e8f39b0eefa6fa9be2ca6a
[   +1 ms]   [        ] 1.26.0-12.0.pre-179-gfc9addb88b
[  +72 ms]   [  +72 ms] executing: [C:\src\flutter_master/] git rev-parse --abbrev-ref --symbolic @{u}
[  +52 ms]   [  +53 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ]   [        ] origin/master
[        ]   [        ] executing: [C:\src\flutter_master/] git ls-remote --get-url origin
[  +48 ms]   [  +50 ms] Exit code 0 from: git ls-remote --get-url origin
[  +11 ms]   [        ] https://github.com/flutter/flutter.git
[  +76 ms]   [  +87 ms] executing: [C:\src\flutter_master/] git rev-parse --abbrev-ref HEAD
[  +53 ms]   [  +53 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ]   [        ] master
[ +129 ms]   [ +130 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +4 ms]   [   +6 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[  +12 ms]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[  +17 ms]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[  +19 ms]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +37 ms]   [  +87 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +1 ms]   [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[  +15 ms]   [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[  +21 ms]   [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[        ]   [        ] Artifact Instance of 'PubDependencies' is not required, skipping update.
[  +87 ms]   [ +134 ms] Initializing file store
[  +26 ms]   [  +26 ms] Done initializing file store
[  +62 ms]   [  +62 ms] Skipping target: gen_localizations
[+2445 ms]   [+2445 ms] Skipping target: unpack_windows
[ +473 ms]   [ +473 ms] Skipping target: kernel_snapshot
[ +326 ms]   [ +326 ms] Skipping target: debug_bundle_windows_assets
[   +1 ms]   [        ] Persisting file store
[  +22 ms]   [  +22 ms] Done persisting file store
[  +12 ms]   [  +12 ms] build succeeded.
[  +11 ms]   [  +11 ms] "flutter assemble" took 3,672ms.
[ +261 ms]   [ +261 ms] ensureAnalyticsSent: 257ms
[   +2 ms]   [   +2 ms] Running shutdown hooks
[  +14 ms]   [        ] Shutdown hooks complete
[   +1 ms]   [   +1 ms] exiting with code 0
[  +90 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\flutter_assemble\flutter_assemble.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\flutter_assemble\flutter_assemble.tlog\flutter_assemble.lastbuildstate".
[   +1 ms] Done Building Project "D:\projects\helloproject\build\windows\flutter\flutter_assemble.vcxproj" (default targets).
[   +6 ms] InitializeBuildStatus:
[  +12 ms]   Creating "flutter_wrapper_app.dir\Debug\flutter_.6E363A59.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[        ] CustomBuild:
[        ]   All outputs are up-to-date.
[  +14 ms] VcpkgTripletSelection:
[        ]   Using triplet "x64-windows" from "C:\src\vcpkg\scripts\buildsystems\msbuild\..\..\..\installed\x64-windows\"
[  +32 ms] ClCompile:
[        ]   All outputs are up-to-date.
[  +28 ms] Lib:
[        ]   All outputs are up-to-date.
[   +1 ms]   flutter_wrapper_app.vcxproj -> D:\projects\helloproject\build\windows\flutter\Debug\flutter_wrapper_app.lib
[  +10 ms] AppLocalFromInstalled:
[  +11 ms]   pwsh.exe -ExecutionPolicy Bypass -noprofile -File "C:\src\vcpkg\scripts\buildsystems\msbuild\applocal.ps1"
"D:\projects\helloproject\build\windows\flutter\Debug\flutter_wrapper_app.lib"
"C:\src\vcpkg\scripts\buildsystems\msbuild\..\..\..\installed\x64-windows\debug\bin"
"flutter_wrapper_app.dir\Debug\flutter_.6E363A59.tlog\flutter_wrapper_app.write.1u.tlog" "flutter_wrapper_app.dir\Debug\vcpkg.applocal.log"      
[ +828 ms] FinalizeBuildStatus:
[   +1 ms]   Deleting file "flutter_wrapper_app.dir\Debug\flutter_.6E363A59.tlog\unsuccessfulbuild".
[        ]   Touching "flutter_wrapper_app.dir\Debug\flutter_.6E363A59.tlog\flutter_wrapper_app.lastbuildstate".
[   +1 ms] Done Building Project "D:\projects\helloproject\build\windows\flutter\flutter_wrapper_app.vcxproj" (default targets).
[ +121 ms] Project "D:\projects\helloproject\build\windows\ALL_BUILD.vcxproj" (3) is building
"D:\projects\helloproject\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (6) on node 1 (default targets).
[   +2 ms] InitializeBuildStatus:
[   +1 ms]   Creating "flutter_wrapper_plugin.dir\Debug\flutter_.AF0C0463.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[   +1 ms] CustomBuild:
[   +2 ms]   All outputs are up-to-date.
[   +2 ms] VcpkgTripletSelection:
[   +1 ms]   Using triplet "x64-windows" from "C:\src\vcpkg\scripts\buildsystems\msbuild\..\..\..\installed\x64-windows\"
[   +5 ms] ClCompile:
[  +21 ms]   All outputs are up-to-date.
[   +1 ms] Lib:
[        ]   All outputs are up-to-date.
[        ]   flutter_wrapper_plugin.vcxproj -> D:\projects\helloproject\build\windows\flutter\Debug\flutter_wrapper_plugin.lib
[        ] AppLocalFromInstalled:
[   +1 ms]   pwsh.exe -ExecutionPolicy Bypass -noprofile -File "C:\src\vcpkg\scripts\buildsystems\msbuild\applocal.ps1"
"D:\projects\helloproject\build\windows\flutter\Debug\flutter_wrapper_plugin.lib"
"C:\src\vcpkg\scripts\buildsystems\msbuild\..\..\..\installed\x64-windows\debug\bin"
"flutter_wrapper_plugin.dir\Debug\flutter_.AF0C0463.tlog\flutter_wrapper_plugin.write.1u.tlog"
"flutter_wrapper_plugin.dir\Debug\vcpkg.applocal.log"
[ +711 ms] FinalizeBuildStatus:
[        ]   Deleting file "flutter_wrapper_plugin.dir\Debug\flutter_.AF0C0463.tlog\unsuccessfulbuild".
[        ]   Touching "flutter_wrapper_plugin.dir\Debug\flutter_.AF0C0463.tlog\flutter_wrapper_plugin.lastbuildstate".
[        ] Done Building Project "D:\projects\helloproject\build\windows\flutter\flutter_wrapper_plugin.vcxproj" (default targets).
[  +41 ms] Project "D:\projects\helloproject\build\windows\ALL_BUILD.vcxproj" (3) is building
"D:\projects\helloproject\build\windows\runner\helloproject.vcxproj" (7) on node 1 (default targets).
[   +6 ms] InitializeBuildStatus:
[        ]   Touching "helloproject.dir\Debug\helloproject.tlog\unsuccessfulbuild".
[   +2 ms] CustomBuild:
[   +3 ms]   All outputs are up-to-date.
[  +73 ms] VcpkgTripletSelection:
[  +23 ms]   Using triplet "x64-windows" from "C:\src\vcpkg\scripts\buildsystems\msbuild\..\..\..\installed\x64-windows\"
[        ] ClCompile:
[        ]   C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.28.29333\bin\HostX64\x64\CL.exe /c
/ID:\projects\helloproject\windows /ID:\projects\helloproject\windows\flutter\ephemeral
/ID:\projects\helloproject\windows\flutter\ephemeral\cpp_client_wrapper\include
/I"C:\src\vcpkg\scripts\buildsystems\msbuild\..\..\..\installed\x64-windows\include" /Zi /nologo /W4 /WX /diagnostics:column /Od /Ob0 /D WIN32 /D_WINDOWS /D _HAS_EXCEPTIONS=0 /D _DEBUG /D NOMINMAX /D UNICODE /D _UNICODE /D "CMAKE_INTDIR=\"Debug\"" /D _UNICODE /D UNICODE /Gm- /EHsc /RTC1   
/MDd /GS /fp:precise /Zc:wchar_t /Zc:forScope /Zc:inline /GR /std:c++17 /Fo"helloproject.dir\Debug\\" /Fd"helloproject.dir\Debug\vc142.pdb" /Gd  
/TP /wd4100 /errorReport:queue D:\projects\helloproject\windows\flutter\generated_plugin_registrant.cc
D:\projects\helloproject\windows\runner\flutter_window.cpp
[  +21 ms]   generated_plugin_registrant.cc
[ +408 ms]   flutter_window.cpp
[ +917 ms]   Generating Code...
[  +88 ms] ResourceCompile:
[        ]   All outputs are up-to-date.
[  +31 ms] Link:
[   +5 ms]   C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.28.29333\bin\HostX64\x64\link.exe /ERRORREPORT:QUEUE/OUT:"D:\projects\helloproject\build\windows\runner\Debug\helloproject.exe" /INCREMENTAL /NOLOGO
/LIBPATH:"C:\src\vcpkg\scripts\buildsystems\msbuild\..\..\..\installed\x64-windows\debug\lib"
/LIBPATH:"C:\src\vcpkg\scripts\buildsystems\msbuild\..\..\..\installed\x64-windows\debug\lib\manual-link"
..\flutter\Debug\flutter_wrapper_app.lib D:\projects\helloproject\windows\flutter\ephemeral\flutter_windows.dll.lib kernel32.lib user32.lib      
gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib
"C:\src\vcpkg\scripts\buildsystems\msbuild\..\..\..\installed\x64-windows\debug\lib\*.lib" /MANIFEST /MANIFESTUAC:"level='asInvoker'
uiAccess='false'" /manifest:embed /manifestinput:D:\projects\helloproject\windows\runner\runner.exe.manifest /DEBUG
/PDB:"D:/projects/helloproject/build/windows/runner/Debug/helloproject.pdb" /SUBSYSTEM:WINDOWS /TLBID:1 /DYNAMICBASE /NXCOMPAT
/IMPLIB:"D:/projects/helloproject/build/windows/runner/Debug/helloproject.lib" /MACHINE:X64  /machine:x64 helloproject.dir\Debug\Runner.res      
[  +21 ms]   helloproject.dir\Debug\flutter_window.obj
[   +1 ms]   helloproject.dir\Debug\main.obj
[        ]   helloproject.dir\Debug\run_loop.obj
[        ]   helloproject.dir\Debug\utils.obj
[        ]   helloproject.dir\Debug\win32_window.obj
[        ]   helloproject.dir\Debug\generated_plugin_registrant.obj
[ +231 ms]   helloproject.vcxproj -> D:\projects\helloproject\build\windows\runner\Debug\helloproject.exe
[   +1 ms] AppLocalFromInstalled:
[  +15 ms]   pwsh.exe -ExecutionPolicy Bypass -noprofile -File "C:\src\vcpkg\scripts\buildsystems\msbuild\applocal.ps1"
"D:\projects\helloproject\build\windows\runner\Debug\helloproject.exe"
"C:\src\vcpkg\scripts\buildsystems\msbuild\..\..\..\installed\x64-windows\debug\bin"
"helloproject.dir\Debug\helloproject.tlog\helloproject.write.1u.tlog" "helloproject.dir\Debug\vcpkg.applocal.log"
[ +661 ms] FinalizeBuildStatus:
[        ]   Deleting file "helloproject.dir\Debug\helloproject.tlog\unsuccessfulbuild".
[        ]   Touching "helloproject.dir\Debug\helloproject.tlog\helloproject.lastbuildstate".
[  +13 ms] Done Building Project "D:\projects\helloproject\build\windows\runner\helloproject.vcxproj" (default targets).
[        ] InitializeBuildStatus:
[   +1 ms]   Creating "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[        ] CustomBuild:
[        ]   All outputs are up-to-date.
[  +14 ms] FinalizeBuildStatus:
[        ]   Deleting file "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\unsuccessfulbuild".
[   +1 ms]   Touching "x64\Debug\ALL_BUILD\ALL_BUILD.tlog\ALL_BUILD.lastbuildstate".
[        ] Done Building Project "D:\projects\helloproject\build\windows\ALL_BUILD.vcxproj" (default targets).
[   +6 ms] InitializeBuildStatus:
[   +8 ms]   Creating "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild" because "AlwaysCreate" was specified.
[  +50 ms] PostBuildEvent:
[        ]   setlocal
[        ]   "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe"    
-DBUILD_TYPE=Debug -P cmake_install.cmake
[  +15 ms]   if %errorlevel% neq 0 goto :cmEnd
[   +1 ms]   :cmEnd
[  +12 ms]   endlocal & call :cmErrorLevel %errorlevel% & goto :cmDone
[        ]   :cmErrorLevel
[   +1 ms]   exit /b %1
[        ]   :cmDone
[        ]   if %errorlevel% neq 0 goto :VCEnd
[        ]   :VCEnd
[  +36 ms]   -- Install configuration: "Debug"
[  +21 ms]   -- Up-to-date: D:/projects/helloproject/build/windows/runner/Debug/data/icudtl.dat
[   +1 ms]   -- Up-to-date: D:/projects/helloproject/build/windows/runner/Debug/flutter_windows.dll
[  +14 ms]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets
[   +8 ms]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/AssetManifest.json
[   +1 ms]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/BalooThambi2-Bold.ttf
[  +12 ms]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/BalooThambi2-ExtraBold.ttf      
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/BalooThambi2-Medium.ttf
[   +1 ms]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/BalooThambi2-Regular.ttf        
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/BalooThambi2-SemiBold.ttf       
[   +1 ms]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/CourierPrime-Bold.ttf
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/CourierPrime-BoldItalic.ttf     
[  +24 ms]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/CourierPrime-Italic.ttf
[   +1 ms]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/CourierPrime-Regular.ttf        
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Manjari-Bold.ttf
[   +1 ms]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Manjari-Regular.ttf
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Manjari-Thin.ttf
[   +1 ms]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Meera-Regular.ttf
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/MeeraInimai-Regular.ttf
[   +1 ms]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Quicksand-Bold.ttf
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Quicksand-Light.ttf
[   +1 ms]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Quicksand-Medium.ttf
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Quicksand-Regular.ttf
[   +1 ms]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Quicksand-SemiBold.ttf
[  +19 ms]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Roboto-Black.ttf
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Roboto-BlackItalic.ttf
[   +1 ms]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Roboto-Bold.ttf
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Roboto-BoldItalic.ttf
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Roboto-Italic.ttf
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Roboto-Light.ttf
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Roboto-LightItalic.ttf
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Roboto-Medium.ttf
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Roboto-MediumItalic.ttf
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Roboto-Regular.ttf
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Roboto-Thin.ttf
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Roboto-ThinItalic.ttf
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/fonts/Uroob-Regular.ttf
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/images
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/images/sccover.png
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/images/sclogo-192.png
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/assets/images/sclogo.png
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/FontManifest.json
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/fonts
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/fonts/MaterialIcons-Regular.otf
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/kernel_blob.bin
[  +63 ms]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/NOTICES
[   +3 ms]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/packages
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons
[        ]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets
[  +14 ms]   -- Installing:
D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/packages/cupertino_icons/assets/CupertinoIcons.ttf
[   +1 ms]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/packages/material_design_icons_flutter       
[  +15 ms]   -- Installing: D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/packages/material_design_icons_flutter/lib
[        ]   -- Installing:
D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/packages/material_design_icons_flutter/lib/fonts
[   +1 ms]   -- Installing:
D:/projects/helloproject/build/windows/runner/Debug/data/flutter_assets/packages/material_design_icons_flutter/lib/fonts/materialdesignicons-webfont.ttf
[  +17 ms] FinalizeBuildStatus:
[  +15 ms]   Deleting file "x64\Debug\INSTALL\INSTALL.tlog\unsuccessfulbuild".
[        ]   Touching "x64\Debug\INSTALL\INSTALL.tlog\INSTALL.lastbuildstate".
[  +12 ms] Done Building Project "D:\projects\helloproject\build\windows\INSTALL.vcxproj" (default targets).
[   +2 ms] Build succeeded.
[   +2 ms]     0 Warning(s)
[  +12 ms]     0 Error(s)
[   +1 ms] Time Elapsed 00:00:12.23
[  +16 ms] Building Windows application... (completed in 13.0s)
[ +298 ms] Observatory URL on device: http://127.0.0.1:55024/wWWgFwxJ8sY=/
[   +4 ms] Caching compiled dill
[  +61 ms] Connecting to service protocol: http://127.0.0.1:55024/wWWgFwxJ8sY=/
[ +178 ms] Launching a Dart Developer Service (DDS) instance at http://127.0.0.1:0, connecting to VM service at
http://127.0.0.1:55024/wWWgFwxJ8sY=/.
[ +120 ms] DDS is listening at http://127.0.0.1:55027/epMCrp4eY_I=/.
[  +76 ms] Successfully connected to service protocol: http://127.0.0.1:55024/wWWgFwxJ8sY=/
[+3215 ms] DevFS: Creating new filesystem on the device (null)
[  +25 ms] DevFS: Created new filesystem on the device (file:///C:/Users/aldri/AppData/Local/Temp/helloprojectafd849ca/helloproject/)
[   +6 ms] Updating assets
[ +131 ms] Syncing files to device Windows...
[   +2 ms] <- reset
[  +12 ms] Compiling dart to kernel with 0 updated files
[   +6 ms] <- recompile package:helloproject/main.dart 87a2c041-af2e-4aa1-9784-4b26a9057f3c
[  +11 ms] <- 87a2c041-af2e-4aa1-9784-4b26a9057f3c
[ +103 ms] Updating files.
[        ] DevFS: Sync finished
[   +1 ms] Syncing files to device Windows... (completed in 137ms)
[  +11 ms] Synced 0.0MB.
[   +1 ms] <- accept
[  +44 ms] Connected to _flutterView/0x17b2eca7350.
[   +2 ms] Flutter run key commands.
[   +2 ms] r Hot reload. 
[   +1 ms] R Hot restart.
[   +1 ms] h Repeat this help message.
[  +11 ms] d Detach (terminate "flutter run" but leave application running).
[   +1 ms] c Clear the screen
[   +1 ms] q Quit (terminate the application on the device).
[        ] An Observatory debugger and profiler on Windows is available at: http://127.0.0.1:55027/epMCrp4eY_I=/
[   +2 ms]
           Flutter DevTools, a Flutter debugger and profiler, on Windows is available at:
           http://127.0.0.1:9106?uri=http%3A%2F%2F127.0.0.1%3A55027%2FepMCrp4eY_I%3D%2F
[  +65 ms] Running with unsound null safety
[   +4 ms] For more information see https://dart.dev/null-safety/unsound-null-safety
[+46084 ms] Service protocol connection closed.
[   +1 ms] Lost connection to device.
[   +5 ms] DevFS: Deleting filesystem on the device (file:///C:/Users/aldri/AppData/Local/Temp/helloprojectafd849ca/helloproject/)
[ +268 ms] Ignored error while cleaning up DevFS: TimeoutException after 0:00:00.250000: Future not completed
[   +3 ms] "flutter run" took 65,729ms.
[ +263 ms] ensureAnalyticsSent: 258ms
[   +1 ms] Running shutdown hooks
[   +1 ms] Shutdown hook priority 4
[  +15 ms] Shutdown hooks complete
[        ] exiting with code 0
```
</details>
<details>
  <summary>Flutter Analyze</summary>

```
flutterm analyze
Analyzing helloproject...                                               
No issues found! (ran in 2.0s)
```
</details>
<details>
  <summary>Flutter Doctor</summary>

```
flutterm doctor -v
[√] Flutter (Channel master, 1.26.0-13.0.pre.179, on Microsoft Windows [Version 10.0.19042.746], locale en-IN)
    • Flutter version 1.26.0-13.0.pre.179 at C:\src\flutter_master
    • Framework revision fc9addb88b (23 hours ago), 2021-01-26 03:24:03 -0500
    • Engine revision f47ab4434d
    • Dart version 2.12.0 (build 2.12.0-257.0.dev)
    • Pub download mirror https://pub.flutter-io.cn

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at C:\Users\aldri\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2019 16.8.4)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    • Visual Studio Community 2019 version 16.8.30907.101
    • Windows 10 SDK version 10.0.18362.0

[√] Android Studio (version 4.1.0)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] VS Code (version 1.52.1)
    • VS Code at C:\Users\aldri\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.18.1

[√] Connected device (4 available)
    • Infinix X660 (mobile) • 052713101N051529 • android-arm64  • Android 10 (API 29)
    • Windows (desktop)     • windows          • windows-x64    • Microsoft Windows [Version 10.0.19042.746]
    • Chrome (web)          • chrome           • web-javascript • Google Chrome 87.0.4280.141
    • Edge (web)            • edge             • web-javascript • Microsoft Edge 88.0.705.50

• No issues found!
```

</details>

**Comment by AldrinMathew** [comment]
**No, this issue is not seen in the DEV and STABLE channels of Flutter**
I have multiple applications under development in various channels of flutter. Only the master channel has this issue.

**Comment by iapicca** [comment]
possibly related to https://github.com/flutter/flutter/issues/72953

**Comment by TahaTesser** [comment]
Hi @AldrinMathew 
Thanks for filing the issue, I was experiencing the issue lately I was about to file this, you beat me to it! 

<details> 
<summary>flutter doctor -v</summary> 

```bash
[✓] Flutter (Channel master, 1.26.0-13.0.pre.194, on Microsoft Windows [Version 10.0.19042.746], locale en-US)
    • Flutter version 1.26.0-13.0.pre.194 at C:\Users\Taha\Code\flutter_master
    • Framework revision 71aec53acb (4 hours ago), 2021-01-26 21:53:04 -0800
    • Engine revision b11bef83a5
    • Dart version 2.12.0 (build 2.12.0-263.0.dev)

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:\Users\Taha\Code\SDK
    • Platform android-30, build-tools 30.0.3
    • ANDROID_HOME = C:\Users\Taha\Code\SDK
    • Java binary at: C:\Users\Taha\Code\android-studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[✓] Visual Studio - develop for Windows (Visual Studio Community 2019 16.8.3)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    • Visual Studio Community 2019 version 16.8.30804.86
    • Windows 10 SDK version 10.0.18362.0

[✓] Android Studio (version 4.1.0)
    • Android Studio at C:\Users\Taha\Code\android-studio
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[✓] VS Code (version 1.52.1)
    • VS Code at C:\Users\Taha\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.18.1

[✓] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19042.746]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 87.0.4280.141
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 88.0.705.50

• No issues found!
```

</details>

Thank you

**Comment by AldrinMathew** [comment]
@TahaTesser
Good to know that I am not alone. But I feel bad that this issue is affecting other's workflows too. Hopefully, it gets resolved soon.

**Comment by ugurcan** [comment]
We are living the same problem in our project at the channel master. 

flutter doctor -v
[√] Flutter (Channel master, 1.26.0-13.0.pre.144, on Microsoft Windows [Version 10.0.18363.1256], locale en-GB)
    • Flutter version 1.26.0-13.0.pre.144 at C:\Src\flutter
    • Framework revision 2007186d2e (3 days ago), 2021-01-24 12:54:00 -0800
    • Engine revision fdddf87080
    • Dart version 2.12.0 (build 2.12.0-253.0.dev)

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.0)
    • Android SDK at C:\Users\ugcan\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.0
    • Java binary at: C:\Program Files\Android\Android Studio1\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Enterprise 2019 16.6.2)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise
    • Visual Studio Enterprise 2019 version 16.6.30204.135
    • Windows 10 SDK version 10.0.18362.0

[√] Android Studio (version 4.0)
    • Android Studio at C:\Program Files\Android\Android Studio1
    • Flutter plugin version 46.0.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] VS Code, 64-bit edition (version 1.52.1)
    • VS Code at C:\Program Files\Microsoft VS Code
    • Flutter extension version 3.18.1

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.18363.1256]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 87.0.4280.141
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 86.0.622.38

• No issues found!

**Comment by AldrinMathew** [comment]
@iapicca
Thanks for mentioning Issue #72953. I checked and unfortunately, it seems like these issues are not related.

1. The Issue mentions that the build method is not triggered when the browser window is minimized, and that it is triggered once the window is maximized. That's not the case here as the window never recovers once it freezes. It freezes instantly at maximizing and also becomes unresponsive just 5 or 6 seconds after maximizing.
2. In both HTML and CanvasKit rendering options for Flutter Web Development, the Debug and Release builds run in-Browser and not like a native application. Plus, Dart is compiled to Javascript before building for web. I could be really wrong here, but as far as I know, these two factors distinguish a Flutter Web Application from a Desktop Application.

**Comment by AldrinMathew** [comment]
@ugurcan
Thanks for attaching the log. Let's hope that this gets resolved soon.

**Comment by stuartmorgan** [comment]
@iskakaushik It sounds like we need to special-case minimizing (which on Windows sets the size to 0x0) in the smooth resize code.

/cc @csells FYI this will need a cherry-pick once fixed.

**Comment by iskakaushik** [comment]
Thanks for finding the bug @AldrinMathew !

**Comment by AldrinMathew** [comment]
Always here to help. Great response time from the contributors and everyone. Keep it up. 👌
💚 from Palakkad, Kerala

**Comment by AldrinMathew** [comment]
@iskakaushik
Thanks a lot for the fix. I will be closing this issue once the checks are done in the PR and it is merged into the branch.

**Comment by alionour** [comment]
I have the same issue !!

**Comment by Bestfastfire** [comment]
I've tested in beta, dev and master and Im having this same issue

**Comment by dru** [comment]
```
Flutter 1.26.0-17.2.pre • channel dev • https://github.com/flutter/flutter.git
Framework • revision 79b49b9e10 (2 days ago) • 2021-02-03 15:33:39 -0800
Engine • revision 2c527d6c7e
Tools • Dart 2.12.0 (build 2.12.0-259.8.beta)
```

App still crashing when restored from minimized state

**Comment by AldrinMathew** [comment]
@stuartmorgan @iskakaushik 
**This issue should be reopened as I ran my application using the Master Channel and the issue reappeared. The version was 1.26.0-18.0.pre.210. I then upgraded to the latest Master Channel and the issue persists.**
``flutterm upgrade`` final output.
```
Flutter 1.26.0-18.0.pre.210 • channel master • https://github.com/flutter/flutter.git
Framework • revision a77388e670 (6 hours ago) • 2021-02-04 22:51:04 -0800
Engine • revision b04955656c
Tools • Dart 2.13.0 (build 2.13.0-0.0.dev)

Running flutter doctor...
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel master, 1.26.0-18.0.pre.210, on Microsoft Windows [Version 10.0.19042.746], locale en-IN)
[√] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
[√] Chrome - develop for the web
[√] Visual Studio - develop for Windows (Visual Studio Community 2019 16.8.4)
[√] Android Studio (version 4.1.0)
[√] VS Code (version 1.53.0)
[√] Connected device (3 available)
```

**Comment by iapicca** [comment]
@TahaTesser 
it would be great if a `test` would be introduce to prevent this issue from re-emerging in the future

**Comment by Bestfastfire** [comment]
With me it happens both in the debug version during codding and in the release. It is simple to reproduce, just run a new project that is, minimize and maximize (both in window mode and full screen) that it already freezes and stops working.

**Comment by TahaTesser** [comment]
Thank you, everyone, I can reproduce it on the latest `dev` and `master` as well. 
Not sure if it is regression or fix has been reverted from I will let the team decide what happened here but IIn either case a test could be interoduced to prevent it from happening again as @iapicca mentioned -> https://github.com/flutter/flutter/issues/74797#issuecomment-774012409. 

<details>
<summary>flutter doctor -v</summary>

```bash
[âˆš] Flutter (Channel dev, 1.26.0-17.2.pre, on Microsoft Windows [Version 10.0.19042.746], locale en-US)
    â€¢ Flutter version 1.26.0-17.2.pre at C:\Users\Taha\Code\flutter_dev
    â€¢ Framework revision 79b49b9e10 (2 days ago), 2021-02-03 15:33:39 -0800
    â€¢ Engine revision 2c527d6c7e
    â€¢ Dart version 2.12.0 (build 2.12.0-259.8.beta)
[âˆš] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    â€¢ Android SDK at C:\Users\Taha\Code\SDK
    â€¢ Platform android-30, build-tools 30.0.3
    â€¢ ANDROID_HOME = C:\Users\Taha\Code\SDK
    â€¢ Java binary at: C:\Users\Taha\Code\android-studio\jre\bin\java
    â€¢ Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    â€¢ All Android licenses accepted.
[âˆš] Chrome - develop for the web
    â€¢ Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe
[âˆš] Visual Studio - develop for Windows (Visual Studio Community 2019 16.8.3)
    â€¢ Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    â€¢ Visual Studio Community 2019 version 16.8.30804.86
    â€¢ Windows 10 SDK version 10.0.18362.0
[âˆš] Android Studio (version 4.1.0)
    â€¢ Android Studio at C:\Users\Taha\Code\android-studio
    â€¢ Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    â€¢ Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    â€¢ Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
[âˆš] VS Code (version 1.53.0)
    â€¢ VS Code at C:\Users\Taha\AppData\Local\Programs\Microsoft VS Code
    â€¢ Flutter extension version 3.19.0
[âˆš] Connected device (3 available)
    â€¢ Windows (desktop) â€¢ windows â€¢ windows-x64    â€¢ Microsoft Windows [Version 10.0.19042.746]
    â€¢ Chrome (web)      â€¢ chrome  â€¢ web-javascript â€¢ Google Chrome 88.0.4324.146
    â€¢ Edge (web)        â€¢ edge    â€¢ web-javascript â€¢ Microsoft Edge 88.0.705.53
â€¢ No issues found!
```
```bash
[âˆš] Flutter (Channel master, 1.26.0-18.0.pre.210, on Microsoft Windows [Version 10.0.19042.746], locale en-US)
    â€¢ Flutter version 1.26.0-18.0.pre.210 at C:\Users\Taha\Code\flutter_master
    â€¢ Framework revision a77388e670 (6 hours ago), 2021-02-04 22:51:04 -0800
    â€¢ Engine revision b04955656c
    â€¢ Dart version 2.13.0 (build 2.13.0-0.0.dev)
[âˆš] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    â€¢ Android SDK at C:\Users\Taha\Code\SDK
    â€¢ Platform android-30, build-tools 30.0.3
    â€¢ ANDROID_HOME = C:\Users\Taha\Code\SDK
    â€¢ Java binary at: C:\Users\Taha\Code\android-studio\jre\bin\java
    â€¢ Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    â€¢ All Android licenses accepted.
[âˆš] Chrome - develop for the web
    â€¢ Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe
[âˆš] Visual Studio - develop for Windows (Visual Studio Community 2019 16.8.3)
    â€¢ Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    â€¢ Visual Studio Community 2019 version 16.8.30804.86
    â€¢ Windows 10 SDK version 10.0.18362.0
[âˆš] Android Studio (version 4.1.0)
    â€¢ Android Studio at C:\Users\Taha\Code\android-studio
    â€¢ Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    â€¢ Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    â€¢ Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
[âˆš] VS Code (version 1.53.0)
    â€¢ VS Code at C:\Users\Taha\AppData\Local\Programs\Microsoft VS Code
    â€¢ Flutter extension version 3.19.0
[âˆš] Connected device (3 available)
    â€¢ Windows (desktop) â€¢ windows â€¢ windows-x64    â€¢ Microsoft Windows [Version 10.0.19042.746]
    â€¢ Chrome (web)      â€¢ chrome  â€¢ web-javascript â€¢ Google Chrome 88.0.4324.146
    â€¢ Edge (web)        â€¢ edge    â€¢ web-javascript â€¢ Microsoft Edge 88.0.705.53
â€¢ No issues found!
```

</details>

Thank you

**Comment by iskakaushik** [comment]
Thanks. I'm taking a look now.

**Comment by stuartmorgan** [comment]
> a test could be introduced to prevent it from happening again

We don't currently have a framework for writing native UI tests. It may be possible to test this with an integration test that simulates the relevant WM_* message though.

**Comment by iskakaushik** [comment]
Created https://github.com/flutter/flutter/issues/75523 to add a test as it is involved and this fix needs to be cherry-picked relatively soon. The fix for this at: https://github.com/flutter/engine/pull/24236. I am continuing to work on the test.

**Comment by pcsosinski** [comment]
anything left to do here or can we mark this issue closed?

**Comment by AldrinMathew** [comment]
@pcsosinski
I think this should be left open as the test has not yet been implemented, and there a lot of users still experiencing this issue.
And it is not clear to me how this is related to #24236

**Comment by pcsosinski** [comment]
thanks @AldrinMathew.  test i believe is tracked in https://github.com/flutter/flutter/issues/75523.  should've been https://github.com/flutter/engine/pull/24236

**Comment by AldrinMathew** [comment]
Oh ok. But is the flutter/engine#24236 commit rolled into the flutter master channel? I upgraded to the latest master channel and the issue persists. It will take time for the dev, beta and stable branches to include the changes. May be keep the issue open until the changes are reflected at least in the master channel?

**Comment by lord-tx** [comment]
I found a little _workaround_ for devs who need to continuously test before the implementation of the fix.

Maximize the application window to full-screen immediately it displays on screen and then you can minimize to taskbar. I have only tested this on Windows 10 Pro (2004 - 19041.746). Hopefully, it's not version dependent.

**Comment by pcsosinski** [comment]
closing because it looks like the only thing left to do is CP the 2nd fix into 1.26, which the cp label covers.

**Comment by sg185180** [comment]
The problem is still seen with the latest Flutter Dev version 1.27.0-1.0.pre. Haven't seen this issue with Dev version 1.26.0-8.0.pre though.

**Comment by AldrinMathew** [comment]
@sg185180 
Yes, it will take some time for the changes to reflect in the `dev` channel. Only tested and relatively stable changes and commits are pushed from the `master` branch to the `dev` branch. So that's why.
